HyperV/scripts/utils.ps1

```diff
@@ -47,7 +48,7 @@ function GitClonePull($path, $url, $branch="master")
         try
         {
             ExecRetry {
-                Remove-Item -Force -Recurse -ErrorAction SilentlyContinue "$path\*"
+                Remove-Item -Force -Recurse "$path\*"
```

Comment:
-ErrorAction Continue
You don't want to force the script to die there, especially in an ExecRetry loop...